src/components/App/App.js

```diff
@@ -1,48 +1,89 @@
-import React, { Component } from 'react'
-import { Link } from 'react-router-dom'
-import classnames from 'classnames'
-import './style.css'
+import React, { Component } from "react";
+import { Link } from "react-router-dom";
+import classnames from "classnames";
+import "./style.css";
```

Comment:
Are the double quote changes important?

Reply:
No, just my Prettier defaults (apologies, it also reformatted a fair bit). I think we can add a .prettierrc that handles formatting per project. Will add.